openfe/tests/setup/test_atommapping.py

```diff
@@ -29,3 +32,34 @@ def test_draw_mapping_svg(tmpdir, other_mapping):
         other_mapping.draw_to_file('test.svg', d2d=d2d)
         filed = pathlib.Path('test.svg')
         assert filed.exists()
+
+
+class TestLigandAtomMappingSerialization:
```

Comment:
Pretty much a direct translation of https://github.com/OpenFreeEnergy/gufe/pull/16 because why make our life difficult 🤷🏽

Reply:
I guess this depends on https://github.com/OpenFreeEnergy/gufe/pull/16 now